src/TreeHouse/CS/Config.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TreeHouse\CS;
+
+use PhpCsFixer\Config as BaseConfig;
+
+class Config extends BaseConfig
+{
+    /**
+     * @param string $name
+     */
+    public function __construct($name = 'treehouse')
+    {
+        parent::__construct($name);
+
+        $this->setRiskyAllowed(true);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function getRules(): array
+    {
+        return [
+            '@Symfony' => true,
+            '@Symfony:risky' => true,
+            'align_multiline_comment' => true,
+            'array_syntax' => [
+                'syntax' => 'short',
+            ],
+            'combine_consecutive_unsets' => true,
+            'concat_space' => [
+                'spacing' => 'one',
+            ],
+            'declare_strict_types' => true,
+            'dir_constant' => true,
+            'list_syntax' => [
+                'syntax' => 'short',
+            ],
+            'phpdoc_add_missing_param_annotation' => true,
+            'phpdoc_inline_tag' => false,
+            'phpdoc_order' => true,
+            'ternary_to_null_coalescing' => true,
+            'yoda_style' => false,
```

Comment:
`no_useless_return` and `ordered_imports` may be nice as well? Even better; remove these rules completely as it suggests there is some shared standard (I thought you wanted to do that later, which is a bit confusing).

Reply:
I used a previous php_cs config as starting point, but I forgot `ordered_imports`, added now 👍 

`no_useless_return` wasn't used before. I'd like to use this as a basis/common set of rules. We can debate and vote for certain rules after this is done.